packages/shared/src/index.ts

```diff
@@ -29,3 +29,4 @@ export * from './services';
 export * from './utils/helpers';
 export * from './utils/constants';
 export * from './utils/loader-handler';
+export * from './utils/benchmark-handler';
```

Comment:
@amir-deriv please migrate this TS instead of JS

Reply:
sure